exercises/dominik_arominski/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include <fstream>
+#include <string>
+#include <map>
+#include <vector>
+#include <algorithm>
+
+std::string RemovePunctuation(std::string word) {
```

Comment:
For the functions and methods, we use lower camel case, so `removePunctuation`.
For passing the arguments, anything that is composite object, including strings, should be passed by & or const & to avoid copy. If you pass by &, there is no point to return.

Reply:
Agreed and will take note of that for the future.